script/match.lua

```diff
@@ -21,7 +23,6 @@ function Match:new (o)
     local player = Player:new(o.player_description)
     player.id = #o.entities + 1
     o.entities[player.id] = player
-
```

Comment:
Can you remove these changes? I find that adding empty newlines between code sections helps for clarity.